scripts/insert-script-in-readme.js

```diff
@@ -0,0 +1,21 @@
+const fs = require('fs');
+
+const readmePath = process.argv[2];
+const minifiedCodePath = process.argv[3];
+
+const codeBlockStart = "```js";
+const codeBlockEnd = "```";
```

Comment:
Small thing, consider putting these above the 2 other consts above, and naming them with the "const convention", like -> `CODE_BLOCK_START` / `CODE_BLOCK_END`.
It just fits more here and is the type of variables that would usually be named like these.